x-pack/plugins/cases/server/client/attachments/add.ts

```diff
@@ -105,6 +105,7 @@ async function getSubCase({
         subCaseId: newSubCase.id,
         fields: ['status', 'sub_case'],
         newValue: JSON.stringify({ status: newSubCase.attributes.status }),
+        owner: newSubCase.attributes.owner,
```

Comment:
Adding the `owner` to user actions.

Reply:
If you do end up keeping the owner constant, it's probably worth updating these to use it as well

Reply:
Yeah good point